addons/calendar/static/src/components/activity/activity.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+    <t t-inherit="mail.Activity" t-inherit-mode="extension">
+        <xpath expr="//button[hasclass('o_Activity_editButton')]" position="replace">
```

Comment:
Not a fan of a straight "replace", that limits other module overrides.

I was wondering if we should not instead append a t-if directive to the existing element and append your button afterwards.

What do you think?